packages/framework-core/test/booster-command-dispatcher.test.ts

```diff
@@ -51,7 +52,7 @@ describe('the `BoosterCommandsDispatcher`', () => {
       const config = {
         commandHandlers: {
           UnauthorizedCommand: {
-            authorizedRoles: [Thor],
+            authorizer: BoosterAuthorizer.authorizeRoles.bind(null, [Thor]),
```

Comment:
Why keep using `function.bind` instead of currying the callee?

I know this is a test file, but I see the pattern all over the codebase

Reply:
What do you mean by "currying the callee"? Do you mean to use nested functions?

Reply:
Yep I mean having a `const authorizer = (arg) => BoosterAuthorizer.authorizeRoles([Thor], arg)`.

I'm not sure if we have `fp-ts-std` installed, but also it can be written as:

```ts
curry2(BoosterAuthorizer.authorizeRoles)([Thor])
```

Anyway this is just stylistic, so let's leave this out 😊